src/forms/datepicker/DatePicker.tsx

```diff
@@ -0,0 +1,423 @@
+                                                >
+                                                    <Icon shape="angle" dir={Direction.RIGHT} />
+                                                </button>
+                                            </div>
+                                        </div>
+                                        <table className="calendar-table weekdays">
+                                            <tbody>
+                                                <tr className="calendar-row">
+                                                    {Moment.weekdaysShort(true).map((day: string) => {
+                                                        return (
+                                                            <td key={"day_" + day} className="calendar-cell weekday">
+                                                                {day.substring(0, 1)}
+                                                            </td>
+                                                        );
+                                                    })}
+                                                </tr>
+                                            </tbody>
+                                        </table>
+                                        <table className="calendar-table calendar-dates">
+                                            <tbody>
+                                                {[0, 1, 2, 3, 4, 5].map((row: number) => {
+                                                    return (
+                                                        <tr key={"tr_" + row} className="calendar-row">
+                                                            {[0, 1, 2, 3, 4, 5, 6].map((cell: number) => {
+                                                                const isToday = calendar.isSame(Moment(), "day"); // are we on today?
+                                                                const isSelected = calendar.isSame(moment, "day"); // are we on selected?
+                                                                const isDisabled = !navValue.isSame(calendar, "month");
+                                                                const day = (
+                                                                    <td key={"td_" + cell} className="calendar-cell">
+                                                                        <div className="day">
+                                                                            <button
+                                                                                type="button"
+                                                                                className={classNames([
+                                                                                    "day-btn", // prettier
+                                                                                    isSelected &&
```

Comment:
As a suggestion I would move out this conditions to somewhere to make cleaner code and more maintainable